internal/blockchain/upgrade.go

```diff
@@ -5928,5 +6083,15 @@ func upgradeUtxoDb(ctx context.Context, db database.DB, utxoBackend UtxoBackend)
 		}
 	}
 
+	// Update to a version 3 utxo database if needed.  This entails removing all
+	// treasurybase utxos since they are never directly spendable and thus
```

Comment:
These references should be "treasurybase txouts" since they are not utxos which is why they're being removed to begin with.

Reply:
Updated.